bluetooth_hid_subcommands_notes.md

```diff
@@ -54,32 +54,33 @@ If the command is `x11`, it polls the MCU State? Used with IR Camera or NFC?
 
 ### Subcommand 0x03: Set input report mode
 
 One argument:
 
-| Argument # | Remarks                                                                                          |
-|:----------:| ------------------------------------------------------------------------------------------------ |
-|   `00`     | Used with cmd `x11`. Active polling for IR camera data. 0x31 data format must be set first       |
-|   `01`     | Same as `00`                                                                                     |
-|   `02`     | Same as `00`. Active polling mode for IR camera data. For specific IR modes                      |
-|   `23`     | MCU update state report?                                                                         |
-|   `30`     | Standard full mode. Pushes current state @60Hz                                                   |
-|   `31`     | NFC/IR mode. Pushes large packets @60Hz                                                          |
-|   `33`     | Unknown mode.                                                                                    |
-|   `35`     | Unknown mode.                                                                                    |
-|   `3F`     | Simple HID mode. Pushes updates with every button press                                          |
-
-`31` input report has all zeroes for IR/NFC data if a `11` ouput report with subcmd `03 00` or `03 01` or `03 02` was not sent before.
+| Arg #  | Remarks                                                                                          |
+|:------:| ------------------------------------------------------------------------------------------------ |
+|  `x00` | Used with cmd `x11`. Active polling for NFC/IR camera data. 0x31 data format must be set first.  |
+|  `x01` | Same as `00`. Active polling mode for NFC/IR MCU configuration data.                             |
+|  `x02` | Same as `00`. Active polling mode for NFC/IR data and configuration. For specific NFC/IR modes   |
+|  `x03` | Same as `00`. Active polling mode for IR camera data. For specific IR modes                      |
+|  `x23` | MCU update state report?                                                                         |
+|  `x30` | Standard full mode. Pushes current state @60Hz                                                   |
+|  `x31` | NFC/IR mode. Pushes large packets @60Hz                                                          |
+|  `x33` | Unknown mode.                                                                                    |
+|  `x35` | Unknown mode.                                                                                    |
+|  `x3F` | Simple HID mode. Pushes updates with every button press                                          |
+
+`x31` input report has all zeroes for IR/NFC data if a `11` ouput report with subcmd `03 00/01/02/03` was not sent before.
```

Comment:
Might be better to be more general: 0x31 replies with all 0’s unless NFC or IR has been initialized/configured

Reply:
Don't worry there will be a break down when the time comes with all cases we know until now.